### DISCUSSION TITLE
스터디 후기입니다.

### DISCUSSION THREAD

**Discussion by asomi70** [fusiondevkr/pl900#69] [1 Q&A]
아. 제가 마지막일듯 하네요. 
늦었지만 다시 한번 좋은 강의 만들어 주신 두분( Justin You 님 / Mina Jin 님 )에게 감사드립니다.
(09월 06일이.. 허전할것 같습니다 ^^;;)
https://blog.naver.com/asomi/222493936940

**Comment by sesme100** [comment]
후기 감사드립니다!! 다음에는 더욱더 자세하게 설명 드릴게요!! 쉽고 정확하고 한번에 깨우칠 수 있는 강의로 만들어 보겠습니다!! 감사합니다!!
